ProjectDataCore.Data/Structures/Model/Alert/AlertModel.cs

```diff
@@ -0,0 +1,50 @@
+﻿using ProjectDataCore.Data.Services.Alert;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectDataCore.Data.Structures.Model.Alert
+{
+    public class AlertModel : IDisposable
```

Comment:
If you want, I would also consider making an alert builder with a [fluent API](https://www.c-sharpcorner.com/article/creating-a-fluent-api-in-c-sharp-net/)